UI/ui_item.csv

```diff
@@ -1741,15 +1741,15 @@ TitleReset#0,"""Select item to undo change"""
 TitleMigration#0,"""Select Item"""
 MigrationButtonTraceStart#0,"""Confirm"""
 MigrationButtonBack#0,"""Back"""
-NoItemMessage#5,"""外見を変更したい武器をセット"""
+NoItemMessage#5,"""Select the weapon to change the appearance of"""
 NoItemMessageReset#0,"""外見を戻したい武器をセット"""
 TraceCheckTitle#0,"""Weapon Form Change Details"""
-TraceMessage#0,"""<yellow>武器フォーム変更パス：$(0)（現在：$(1)）<br>必要メセタ：$(2) （現在：$(3)）<c>"""
+TraceMessage#0,"""<yellow>Weapon Form Pass Needed: $(0)（Have: $(1)）<br>Meseta Needed: $(2) （Have: $(3)）<c>"""
 TraceCheckSubTitle1#0,"""Weapon Form"""
 TraceCheckSubTitle2#0,"""Required items"""
 TraceCheckSubTitle3#0,"""Weapon Form Change item"""
 ItemTraceCheckTitle#0,"""Confirm Weapon Form"""
-ItemTraceMessage#0,"""武器の外見を変更します。<br>よろしいですか？<br><br>※アイテムの性能は変わりません。<br>※ベースアイテムのフォトンカラーを<br>\u3000変更している場合、外見を元に戻しても<br>\u3000フォトンカラーがデフォルトの色に変更されます。<br>※外見を変更すると<br>\u3000ベースアイテムと外見に使用したアイテム両方が<br>\u3000トレードやマイショップの出品アイテムとして<br>\u3000選択できなくなります。"""
+ItemTraceMessage#0,"""This will change the appearance of the weapon.<br>Continue?<br><br>※The item stats will not change<br>※Photon color information<br>\u3000The photon color will bechanged to the default<br>\u3000even if it was changed.<br>※Once you change the appearance<br>\u3000Both the base item and the item changed<br>\u3000will be become untradeable."""
```

Comment:
Now there a spelling "mistake", it is really "bechanged" or "be changed"?

Reply:
yee, already fixed that in the new push lol